docs/src/main/paradox/bootstrap.md

```diff
@@ -87,74 +87,60 @@ of them should perform this join. We make this decision by sorting the known add
+    - Once the lowest addressed node has joined itself, it has formed a new cluster.
+    - Other nodes, including X, will notice that A has started returning *itself* as a seed-node in the Contact Point responses.
     - Any node, including X, immediately joins such seed node that it has observed in the Contact Point process.
     - Nodes continue probing the other nodes, and eventually will notice any of the existing nodes that are part of the cluster,
       and immediately join it. This phase is referred to as "epidemic joining".
     - Eventually all nodes have joined the same cluster, the process is complete.
-- Case 2) A cluster exists already, and when probing the various nodes node X will receive at least one seed-node address,
-from the contact points. 
-    - The node joins any discovered (via Contact Points probing) seed-node and immediately becomes part of the cluster.
-    - The process is complete, the node has successfully joined the "right" cluster.
-        - Notice that this phase is exactly the same as the "epidemic joining" in the more complicated Case 1 when a new 
-          cluster has to be formed.
-
-Additionally, a setting is provided, `akka.management.cluster.bootstrap.form-new-cluster` that can be disabled to only allow the
-node to join existing clusters (Case 2 above). This can be used to provide additional safety during the typical deployment
-as it is relatively rare that no cluster exists. It can be specified in your `application.conf` or provided as an
-argument to the process itself via `-Dakka.management.cluster.bootstrap.form-new-cluster=<true|false>`. By default, it is enabled.
-
-In the next sections we explain the process in more detail.
-
-### Case 1: "Initial" Bootstrap process
-
-Both cases need to discover neighbouring nodes, which they do via Akka Discovery,
-which we recommend (and implement) with DNS, though you may use alternative lookup methods if you want. The bootstrap 
-process will then start probing those nodes Contact Points (which are HTTP endpoints, exposed via Akka Management by the
-Bootstrap Management Extension) for known seeds to join. Since no cluster exists yet, the seed node replies will be 
-empty from all contact points. Once at least `akka.management.cluster.bootstrap.required-contact-point-nr` nodes are
-discovered and `akka.management.cluster.bootstrap.stable-margin` time has passed, the nodes will decide that it is time 
-to confidently (enough) form a new cluster. The node with the "lowest" contact point address will decide to join itself,
-and other nodes are expecting this awaiting a seed-node address to appear in the contact point probing responses.
-
-Once the lowest addressed node has joined itself, it has formed a new cluster, and therefore starts returning itself
-as seed-node in its Contact Point response. Other nodes notice this and will join it, growing the existing cluster.
```

Comment:
this 3rd repetition of the same thing was too much for my taste, made it more concise hopefully without loosing anything

Reply:
Yes it seems all info is still there, thanks